odl/discr/discretization.py

```diff
@@ -186,6 +183,7 @@ def element(self, inp=None):
         if inp is None:
             return self.element_type(self, self.dspace.element())
         try:
+            # pylint: disable=not-callable
```

Comment:
Dislike these disables. pylint seems to fail us here, both for `element_type` and all `Operator`'s, id say disable globally.